com.woltlab.wcf/templates/boxSignedInAs.tpl

```diff
@@ -1,4 +1,6 @@
 <div class="box96">
+	{event name='signedInAs'}
```

Comment:
This event messes with the `box96` layout, which targets the first child.

Reply:
All the events I have submitted, I would like to have for my "Clantag" plugin, which sets a clantag before or behind the username. In this template the placement of the event does not matter. It could be at <div class="box96"> > <div> for example.